ee/src/main/java/io/supertokens/ee/EEFeatureFlag.java

```diff
@@ -151,6 +151,26 @@ public void syncFeatureFlagWithLicenseKey()
             licenseKey = this.getLicenseKeyFromDb();
             this.isLicenseKeyPresent = true;
         } catch (NoLicenseKeyFoundException ex) {
+            try {
+                // Need to check if multitenancy is enabled on the base app and then report paid usage stats
+                EE_FEATURES[] features = FeatureFlag.getInstance(main, new AppIdentifier(null, null))
+                        .getEnabledFeatures();
+                for (EE_FEATURES feature : features) {
+                    if (feature.equals(EE_FEATURES.MULTI_TENANCY)) {
+                        licenseKey = this.getRootLicenseKeyFromDb();
+                        verifyLicenseKey(licenseKey); // also sends paid user stats for the app
+                        try {
+                            // small delay between license checks so that we have a delay for each license key check calls
+                            Thread.sleep(5);
```

Comment:
why this delay? Makes no sese.

Reply:
Bhumil wanted different timestamps, I guess that already happens because of the network call, so no need for this